packages/stark-ui/src/modules/pretty-print/components/pretty-print.component.spec.ts

```diff
@@ -0,0 +1,706 @@
+		"}",
+		"p{",
+		"  margin-bottom:1em",
+		"}",
+		"ul{",
+		"  margin-left:20px;",
+		"  margin-bottom:1em",
+		"}\n" // an extra line break is added at the end
+	].join("\n"); // should contain line breaks
+
+	const rawScssData: string = [
+		"$font-stack: Helvetica, sans-serif; $primary-color: #333; body { font: 100% $font-stack; color: $primary-color; }"
+	].join("");
+
+	const formattedScssData: string = [
+		"$font-stack: Helvetica, sans-serif;",
+		" $primary-color: #333;",
+		" body {",
+		"   font: 100% $font-stack;",
+		"   color: $primary-color;",
+		"}\n" // an extra line break is added at the end
+	].join("\n"); // should contain line breaks
+
+	const rawSqlData: string = [
+		"SELECT DISTINCT Name FROM Production.Product AS p WHERE EXISTS (SELECT *",
+		"FROM Production.ProductModel AS pm WHERE p.ProductModelID = pm.ProductModelID",
+		"AND pm.Name LIKE 'Long-Sleeve Logo Jersey%')"
+	].join("");
+
+	const formattedSqlData: string = [
+		"SELECT DISTINCT Name",
+		"FROM Production.Product AS p",
+		"WHERE EXISTS ",
+		"  (SELECT *FROM Production.ProductModel AS pm",
+		"  WHERE p.ProductModelID = pm.ProductModelIDAND pm.Name LIKE 'Long-Sleeve Logo Jersey%')"
+	].join("\n"); // should contain line breaks
+
+	const rawJsonData: string = [
+		'{"menu": { "id": "file", "value": "File",',
+		'"menuitem": [{"value": "New", "onclick": "CreateNewDoc()"},',
+		'{"value": "Open", "onclick": "OpenDoc()"},',
+		'{"value": "Close", "onclick": "CloseDoc()"}]}}'
+	].join("");
+
+	const formattedJsonData: string = [
+		"{",
+		'  "menu": {',
+		'    "id": "file",',
+		'    "value": "File",',
+		'    "menuitem": [',
+		"      {",
+		'        "value": "New",',
+		'        "onclick": "CreateNewDoc()"',
+		"      },",
+		"      {",
+		'        "value": "Open",',
+		'        "onclick": "OpenDoc()"',
+		"      },",
+		"      {",
+		'        "value": "Close",',
+		'        "onclick": "CloseDoc()"',
+		"      }",
+		"    ]",
+		"  }",
+		"}"
+	].join("\n"); // should contain line breaks
+
+	const rawJavaScriptData: string = [
+		"function calculateData(seed, operationFn) {" +
+			"var data = operationFn(seed);" +
+			"if (!data){" +
+			"data = 'could not calculate data';" +
+			"}" +
+			"return data;" +
+			"}"
+	].join("");
+
+	const rawTypeScriptData: string = [
+		"function calculateData(seed:any, operationFn:Function):any {" +
+			"var data:any = operationFn(seed);" +
+			"if (!data){" +
+			"data = 'could not calculate data';" +
+			"}" +
+			"return data;" +
+			"}"
+	].join("");
+
+	/**
+	 * async beforeEach
+	 */
+	beforeEach(async(() => {
+		return TestBed.configureTestingModule({
+			declarations: [StarkPrettyPrintComponent, TestHostComponent],
+			providers: [
+				{ provide: STARK_LOGGING_SERVICE, useValue: new MockStarkLoggingService() },
+				{ provide: STARK_ROUTING_SERVICE, useClass: MockStarkRoutingService }
+			],
+			schemas: [NO_ERRORS_SCHEMA] // tells the Angular compiler to ignore unrecognized elements and attributes
```

Comment:
Please add to the comment which elements are the ones we don't want to include in the TestingModule